ProjectTemplates/templates/ComponentsWebAssembly-CSharp/.template.config/template.json

```diff
@@ -211,6 +211,51 @@
             "Server/Controllers/OidcConfigurationController.cs",
             "Server/Models/ApplicationUser.cs"
           ]
+        },
+        {
+          "condition": "(!GenerateApi)",
+          "exclude": [
+            "Client/Pages/CallWebApi.razor",
+            "Server/Controllers/CallWebApiController.cs"
+          ]
+        },
+        {
+          "condition": "(!GenerateGraph)",
+          "exclude": [
+            "Client/Shared/NavMenu.CallsMicrosoftGraph.razor",
+            "Client/Pages/ShowProfile.razor",
+            "Server/Controllers/ShowProfileController.cs"
+            ]
+        },
+        {
+          "condition": "(!GenerateApiOrGraph)",
```

Comment:
For the renaming, we need to change this condition into two (because the layout on disk is different when not-hosted (all flat), and hosted (sub folder client, server, etc ...)

![image](https://user-images.githubusercontent.com/13203188/94696013-e3ce5680-0336-11eb-91b9-caf9f5e945ea.png)


```json
        {
          "condition": "(!GenerateApiOrGraph && Hosted)",
          "rename": {
            "Client/Shared/NavMenu.NoGraphOrApi.razor": "Client/Shared/NavMenu.razor"
          },
          "exclude": [
            "Client/Shared/NavMenu.CallsMicrosoftGraph.razor",
            "Client/Shared/NavMenu.CallsWebApi.razor"
          ]
        },
        {
          "condition": "(!GenerateApiOrGraph && !Hosted)",
          "rename": {
            "Client/Shared/NavMenu.NoGraphOrApi.razor": "Shared/NavMenu.razor"
          },
          "exclude": [
            "Client/Shared/NavMenu.CallsMicrosoftGraph.razor",
            "Client/Shared/NavMenu.CallsWebApi.razor"
          ]
        },
```
```

Reply:
got it. thanks.